packages/charts/src/charts/heatmap/heatmap-chart.component.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { FC, useRef } from 'react';
+import { motion, AnimatePresence } from 'framer-motion';
+import { Layout } from '@keen.io/ui-core';
+
+import { generateBlocks } from './heatmap-chart.utils';
+
+import Heatmap from './heatmap.component';
+// import Tooltip from './tooltip.component';
+
+import { Tooltip } from '@keen.io/ui-core';
+import TooltipContent from './tooltip-content.component';
+import { useTooltip } from '../../hooks';
+
+import ShadowFilter from './shadow-filter.component';
+import { ChartBase, /*ChartTooltip,*/ Axes } from '../../components';
+import { margins as defaultMargins, theme as defaultTheme } from '../../theme';
+
+import { ScaleSettings, CommonChartSettings } from '../../types';
+
+export type Props = {
+  /** chart data */
+  data: object[];
+  /** Name of data object property used to create labels on axis */
+  labelSelector: string;
+  /** Minimum value for axis */
+  minValue?: number | 'auto';
+  /** Maximum value for axis */
+  maxValue?: number | 'auto';
+  /** Keys picked from data object used to generate lines */
+  keys?: string[];
+  /** Keys that are disabled for rendering data series */
+  disabledKeys?: string[];
+  /** Layout applied on chart bars */
+  layout?: Layout;
+  /** X scale settings */
+  xScaleSettings?: ScaleSettings;
+  /** Y scale settings */
+  yScaleSettings?: ScaleSettings;
+  /** Block padding */
+  padding?: number;
+} & CommonChartSettings;
+
+export const LineChart: FC<Props> = ({
+  data,
+  svgDimensions,
+  labelSelector,
+  theme = defaultTheme,
+  margins = defaultMargins,
+  minValue = 'auto',
+  maxValue = 'auto',
+  keys = ['value'],
+  disabledKeys = [],
+  layout = 'vertical',
+  xScaleSettings = { type: 'band' },
+  yScaleSettings = { type: 'band' },
+  padding = 2,
+}) => {
+  const { blocks, xScale, yScale, select } = generateBlocks({
```

Comment:
The `select` property is only imported in utils then passed down throught few components - to be used in `Block` - `select(element.current).raise();`.

I think we could simplify it - as the upper tree components do not need to know about that function and import d3-select just in Block component.